installation/scripts/testing.sh

```diff
@@ -26,12 +26,15 @@ metadata:
   name: ${suiteName}
 spec:
   maxRetries: 1
-  concurrency: 1
+  concurrency: 2
 EOF
 
 startTime=$(date +%s)
 
 testExitCode=0
+minCache=-1
```

Comment:
minCache is a very enigmatic name for me

Reply:
changed to `previousPrintTime`